sdks/typescript/featurehub-javascript-client-sdk/README.md

```diff
@@ -18,6 +18,10 @@ We have deprecated [FeatureHub Eventsource Javascript SDK](https://www.npmjs.com
 ## Changelog
 
 ### featurehub-javascript-client-sdk
+#### 1.0.5
+- A case existed in the Polling Client where the client requests could cascade and end up DDOSing the server. On
```

Comment:
Could we keep the release notes tidy and say just this "Fix issue with the polling client (link: https://github.com/featurehub-io/featurehub/issues/465")